modules/verifying-serverless-app-deployment.adoc

```diff
@@ -24,3 +31,8 @@ helloworld-go   http://helloworld-go.default.example.com   helloworld-go-4wsd2
 $ curl http://helloworld-go.default.example.com
 Hello World: Go Sample v1!
 ----
++
+----
+$ curl https://helloworld-go.default.example.com
+Hello World: Go Sample v1!
+----
```

Comment:
Thanks. I think it looks much better now. But I've just realised that for completeness we should state something like this:
Use the `--insecure` parameter with the curl command in order to ignore the error with self signed certificate in certificate chain. If your OpenShift cluster is configured with a certificate that is signed by a certificate authority (CA) make sure to use it with the curl command.

Reply:
I made the text above more explicit. I'd suggest using this or similar. And I'd probably avoid putting the whole certificate error in the docs.